src/main/java/seedu/address/model/statistics/Statistics.java

```diff
@@ -0,0 +1,134 @@
+package seedu.address.model.statistics;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.internship.InternshipApplication;
+import seedu.address.model.status.Status;
+
+/**
+ * Represents a statistics generator model.
+ * Generates relevant statistics for internship applications.
+ */
+public class Statistics {
+    public static final String TOTAL = "TOTAL";
+    public static final double TOTAL_PERCENTAGE = 100;
+
+    private int wishlistCount = 0;
+    private int appliedCount = 0;
+    private int interviewCount = 0;
+    private int offeredCount = 0;
+    private int rejectedCount = 0;
+    private int totalCount = 0;
```

Comment:
Instead of manually listing out all the different statuses, you can instead try something like:
```
Status[] statuses = Status.class.getEnumConstants();
HashMap<Status, Integer> statusCounts = new HashMap<>;
for (status : statuses){statusCounts .add(status, 0)}

```

Reply:
Thanks for the advice! This is a very neat implementation.